fido-key-manager/README.md

```diff
@@ -0,0 +1,135 @@
+Start-Process "shell:AppsFolder\$((Get-StartApps Terminal | Select-Object -First 1).AppId)" -Verb RunAs
+# B) run PowerShell as Administrator.
+Start-Process -FilePath "powershell" -Verb RunAs
+
+# Run fido-key-manager from the Administrator terminal:
+.\target\debug\fido-key-manager.exe --help
+```
+
+## Commands
+
+**Important:** to prevent accidents, make sure you have only **one**
+authenticator connected to your computer when running `fido-key-manager`.
+
+More information about the commands listed below can be seen by running
+`fido-key-manager --help` or `fido-key-manager [command] --help`. Unless
+otherwise specified below, all commands require an authenticator which supports
+*at least* CTAP 2.0.
+
+Command | Description | Requirements
+------- | ----------- | ------------
+`info` | get information about connected authenticators
+`selection` | request user presence on a connected authenticator | CTAP 2.1
+`set-pin` | sets a PIN on an authenticator which doesn't already have a PIN set
+`change-pin` | changes a PIN on an authenticator which has a PIN set
+`factory-reset` | resets an authenticators to factory defaults, deleting all key material
+`enable-enterprise-attestation` | enables the [Enterprise Attestation][] feature | CTAP 2.1
+`set-pin-policy` | set a [Minimum PIN Length][] policy, or force a PIN change before next use | CTAP 2.1
+`toggle-always-uv` | toggles the [Always Require User Verification][] feature | CTAP 2.1
+`bio-info` | shows information about an authenticator's fingerprint sensor | CTAP 2.1-PRE, fingerprint sensor
+`list-fingerprints` | lists enrolled fingerprints | CTAP 2.1-PRE, fingerprint sensor
+`enroll-fingerprint` | enroll a new fingerprint | CTAP 2.1-PRE, fingerprint sensor
+`rename-fingerprint` | renames an enrolled fingerprint | CTAP 2.1-PRE, fingerprint sensor
+`remove-fingerprint` | removes an enrolled fingerprint | CTAP 2.1-PRE, fingerprint sensor
+
+[Always Require User Verification]: https://fidoalliance.org/specs/fido-v2.1-ps-20210615/fido-client-to-authenticator-protocol-v2.1-ps-errata-20220621.html#sctn-feature-descriptions-alwaysUv
+[Enterprise Attestation]: https://fidoalliance.org/specs/fido-v2.1-ps-20210615/fido-client-to-authenticator-protocol-v2.1-ps-errata-20220621.html#sctn-feature-descriptions-enterp-attstn
+[Minimum PIN Length]: https://fidoalliance.org/specs/fido-v2.1-ps-20210615/fido-client-to-authenticator-protocol-v2.1-ps-errata-20220621.html#sctn-feature-descriptions-minPinLength
+
+## Platform-specific notes
+
+### Linux
+
+* NFC support requires [PC/SC Lite][], and a PC/SC initiator (driver) for your
+  NFC transceiver (reader).
+
+  If you're using a transceiver with an NXP PN53x-series chipset (eg: ACS
+  ACR122, Sony PaSoRi), you will need to block the `pn533` and `pn533_usb`
+  kernel modules (which are incompatible [all other NFC software][linuxnfc])
+  from loading:
+
+  ```sh
+  echo "blacklist pn533" | sudo tee -a /etc/modprobe.d/blacklist.conf
```

Comment:
Might be good to name the modprobe.d file differently here for making it clear that it's nfc or webauthn related.

Reply:
Done